src/validators/function.rs

```diff
@@ -262,9 +262,36 @@ pub struct FunctionWrapValidator {
     name: String,
     is_field_validator: bool,
     info_arg: bool,
+    hide_input_in_errors: bool,
 }
 
-impl_build!(FunctionWrapValidator, "function-wrap");
+impl BuildValidator for FunctionWrapValidator {
```

Comment:
Had to implement this for `FunctionWrapValidator` because the above macro was used by `before`, `after` and `wrap` validator but the first two don't have `hide_input_in_errors` and it makes lint failed because of unused variable